### DISCUSSION TITLE
Not able to add Qubino 3-Phase Smart Meter

### DISCUSSION THREAD

**Discussion by sywor** [zwave-js/zwave-js-ui#2741] [Q&A]
I'm trying to add a Qubino 3-Phase Smart Meter to my Z-Wave network. I'm using Home Assistant and a Z-Stick 7 from Aeotec.
The Aeotec stick show up in the UI with the right frequency just fine.  

![image](https://user-images.githubusercontent.com/1451737/196262079-7569e3b8-4308-430d-8417-c720d541ce1f.png)

I have tried adding it using the "Default" and the "Scan QR code" method. none of them work.
After scanning the QR code it shows up under Provisioning Entities

![image](https://user-images.githubusercontent.com/1451737/196275726-9e3a146a-dac7-4ff8-8cda-7a6320718807.png)

Anyone here had any luck adding this kind of device?

**Comment by robertsLando** [comment]
> I'm trying to add a Qubino 3-Phase Smart Meter to my Z-Wave network

Was it previously bounded to anothere network? Did you try to reset your device? If it is in Provisioning entries table it should be added automatically once controller see it, are them close?

**Comment by sywor** [comment]
It's brand new and I have reset it several times. The distance between the Smart Meter and the Aeotec stick is approximately 2 meters with no obstructions other then the plastic lid of my fuse box.  

Could it be that the Aeotec stick is not working correctly? I read somewhere that it's hardware optimized for certain frequency ranges and it doesn't like changing region. Its original region was US.

**Reply by robertsLando** [reply]
> Could it be that the Aeotec stick is not working correctly? I read somewhere that it's hardware optimized for certain frequency ranges and it doesn't like changing region. Its original region was US.

It could be that, sure, I also suggest to try to use an usb ectension cable to see if this fixes the problem.

cc @AlCalzone @kpine

**Reply by AlCalzone** [reply]
Apologies for the delay. If this is still happening, I suggest upgrading the stick's firmware to version 7.17.2. Everything below that version needs to be considered broken. Above too I'd say, but not as badly.

Aeotec have it here for download: https://aeotec.freshdesk.com/support/solutions/articles/6000252296-update-z-stick-7-with-windows
